### PR TITLE
Fix copy of icon images to init project

### PR DESCRIPTION
**Pull request by davrodpin**

### CODE REVIEW

File: bin/grommet.js [hunk 1]
```diff
@@ -43,8 +43,8 @@ var tasks = {
         process.chdir('./' + app);
 
         var templateFolder = path.join(grommetPath, 'templates/init/**');
-        var mobileIcon = path.join(grommetPath, 'mobile-app-icon.png');
-        var shortcutIcon = path.join(grommetPath, 'shortcut-icon.png');
+        var mobileIcon = path.join(grommetPath, 'src/img/mobile-app-icon.png');
```
Comment (alansouzati) [inline bin/grommet.js]: I believe this should be `img/mobile-app-icon.png`. When we install grommet, there is no `src` anymore.

File: bin/grommet.js [hunk 2]
```diff
@@ -43,8 +43,8 @@ var tasks = {
...
+        var shortcutIcon = path.join(grommetPath, 'src/img/shortcut-icon.png');
```
Comment (alansouzati) [inline bin/grommet.js]: same here.

File: bin/grommet.js (hunk 1 above)
Comment (davrodpin) [inline bin/grommet.js]: > I believe this should be img/mobile-app-icon.png. When we install grommet, there is no src anymore.

After run *grommet init*, all source files should go the target *src* directory (e.g. *app-name/src/*), so the *index.html* located under *app-name/src/index.html* can reach them. Isn't that right?

$ tree app-name/
app-name/
├── customEslintrc.json
├── grommet-toolbox.config.js
├── gulpfile.babel.js
├── package.json
└── src
    ├── img
    │   ├── mobile-app-icon.png
    │   └── shortcut-icon.png
    ├── index.html
    ├── js
    │   ├── components
    │   │   └── TodoAppDashboard.js
    │   └── index.js
    └── scss
        └── index.scss

File: bin/grommet.js [hunk 3]
```diff
@@ -43,8 +43,8 @@ var tasks = {
...
 
         gulp.src(mobileIcon).pipe(gulp.dest('./src/img'));
```
Comment (alansouzati) [inline bin/grommet.js]: I believe here is the destination folder you can see we use `src/img` here.

But the place where we grab the image from should be `path.join(grommetPath, 'img/shortcut-icon.png');`

File: bin/grommet.js (hunk 1 above)
Comment (alansouzati) [inline bin/grommet.js]: screen shot 2016-06-27 at 5 47 08 pm

this is the content of the install grommet in your node_modules folder.

**Comment by alansouzati** [comment]
thanks for your contribution

**Pull request merged**